contracts/src/Utilities/ValidatorSetSig.sol

```diff
@@ -0,0 +1,137 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import {
+    WarpMessage,
+    IWarpMessenger
+} from "@avalabs/subnet-evm-contracts@1.2.0/contracts/interfaces/IWarpMessenger.sol";
+import {ReentrancyGuard} from "@openzeppelin/contracts@4.8.1/security/ReentrancyGuard.sol";
+
+/**
+ * THIS IS AN EXAMPLE CONTRACT THAT USES UN-AUDITED CODE.
+ * DO NOT USE THIS CODE IN PRODUCTION.
+ */
+
+/**
+ * @dev Message format for the WarpMessage payload to be forwarded to the target contract
+ *
+ * targetBlockchainID: Blockchain ID of the chain the message is intended for
```

Comment:
we tend to use `@param` even on struct variables

Reply:
But what's the benefit? If it's just style preference that's fine but if future solidity version provides a way to document start fields in NatSpec It's very unlikely to be `@param` and we would have to change it then again.

Reply:
better readability, keeping to a consistent style throughout the repo

Reply:
Looks like we don't use `@param` anywhere in this repo yet. Since there is no special handling for this in terms of documentation generation and `@param` is a misnomer, what do people think about using `@field` or `@member` instead if we do want to prefix them?